docs/src/_includes/components/version-switcher.html

```diff
@@ -1,7 +1,7 @@
 <div class="version-switcher">
-    <a href="/versions/" class="switcher-fallback">Versions</a>
+    <a hidden href="/versions/" class="switcher-fallback">Versions</a>
 
-    <div hidden role="region" class="switcher switcher--version" aria-labelledby="version-switcher-label" id="version-switcher">
+    <div  role="region" class="switcher switcher--version" aria-labelledby="version-switcher-label" id="version-switcher">
```

Comment:
The fallback link is for users who have disabled JavaScript in their browsers. We have a [script](https://github.com/eslint/eslint/blob/10249ad1a961463b6b347be71c074951ab8e2652/docs/src/assets/js/main.js#L131-L154) that hides fallback links and unhides select elements that need JavaScript to work (execution of the script by itself means that JavaScript is enabled).

If the fallback link is initially hidden and the select element is not, these users will see only the select element which doesn't work without JavaScript.

Reply:
oh okay 👍  I should have checked it.